shenyu-admin/src/main/resources/application.yml

```diff
@@ -88,25 +87,9 @@ shenyu:
   jwt:
     key: 2095132720951327
   shiro:
-    white-list:
-      - /
-      - /favicon.*
-      - /static/**
-      - /index**
-      - /plugin
-      - /platform/**
-      - /websocket
-      - /configs/**
-      - /shenyu-client/**
-      - /error
-      - /actuator/health
-      - /swagger-ui.html
-      - /webjars/**
-      - /swagger-resources/**
-      - /v2/api-docs
-      - /csrf
+    white-list: /,/favicon.*,/static/**,/index**,/plugin,/platform/**,/websocket,/configs/**,/shenyu-client/**,/error,/actuator/health
```

Comment:
not modify this

Reply:
I'm not modify this, i just merge master.

Reply:
I made an error while merging,